### ISSUE TITLE
Attribute embedded-clock(leaf-list) needs to be included to sync-model

### ISSUE THREAD

**Issue by PrathibaJee** [openBackhaul/synchronization#7]
As a consequence of the [coreModel issue#22](https://github.com/openBackhaul/core/issues/22) , we require to include the leaf-list embedded-clock in sync-model and augment it to the logical-termination-point in core-model. 
Sharing a draft of the resulting yang , 

```
augment "/core-model:control-construct/core-model:logical-termination-point" {
    uses embedded-clock-spec;  
    description
      "none";
  }
grouping embedded-clock-spec {
    leaf-list embedded-clock {
      type leafref {
        path "/core-model:control-construct/sync-model:clock-collection/sync-model:clock/sync-model:local-id";
        require-instance false;
      }
      description
        "none";
    }
    description
      "none";
  }
```

**Comment by openBackhaul** [comment]
**Informal decision made during the 5G-xhaul call on 26th of January 2022:**
The augments proposed by Prathiba above shall be part of the initial delivery of the Synchronization YANG, which is to be reviewed by the group members in total afterwards.

**Comment by openBackhaul** [comment]
Should be covered by EmbeddedClockSpec class containing _embeddedClock : Clock [*] attribute.

**Comment by openBackhaul** [comment]
It seems SynchLtpSpec and EmbeddedClockSpec were actually redundant and deleting EmbeddedClockSpec was fine.

**Comment by demx8as6** [comment]
Expectation on YANG: "_embeddedClock" attributed is added by augmentation of SynchLtpSpec to the CoreModel:LTP.

**Comment by PrathibaJee** [comment]
yes , the [comment](https://github.com/openBackhaul/synchronization/issues/7#issuecomment-1090004302) is correct.

**Reason :**
  In the current CoreModel , the following 2 attributes refers embeddedClock  , 
  1. phase-aligned-clock
  2. encompassed-clock
  
  As per the [issue ](https://github.com/openBackhaul/core/issues/22), this two attributes will be pruned from CoreModel and added to SyncModel. 
  So pruning the embeddedClock  attribute from CoreModel will be fine.
